.github/workflows/clt_nightly.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         image: [ "ubuntu:bionic", "ubuntu:focal", "ubuntu:jammy", "debian:buster", "debian:bullseye", "debian:bookworm" ]
     runs-on: ubuntu-22.04
     steps:
-      - uses: manticoresoftware/clt@0.1.13
+      - uses: manticoresoftware/clt@0.1.15
```

Comment:
We already use in the same pull 0.1.17, we should use consistent version across the whole workflow